depends/packages/fontconfig.mk

```diff
@@ -11,6 +11,7 @@ define $(package)_set_vars
 endef
 
 define $(package)_config_cmds
+  export PKG_CONFIG_PATH=$(host_prefix)/lib/pkgconfig && \
```

Comment:
@theuni - When you get a chance, could you please comment on [my comment above](https://github.com/etotheipi/BitcoinArmory/pull/304?_pjax=%23js-repo-pjax-container#discussion_r36862489) about the PKG_CONFIG env vars seeming to be needed for RasPi depends builds?

That appears to be the last outstanding issue from your review. Thanks again for the review.